content/tutorials/trainapp-fitbit.textile

```diff
@@ -0,0 +1,550 @@
+index: 95
+---
+
+In this tutorial, we will see how to use the "Ably Realtime client library":/realtime to build a realtime London Tube schedule clockface app for Fitbit. Ably enables realtime data sharing using Pub/Sub messaging architecture via a concept called "channels":https://www.ably.io/channels.
+
+For the purpose of this tutorial, we'll not discuss storage of messages or other server-side mechanisms.
+
+
+"Getting started with Fitbit SDK":https://dev.fitbit.com/getting-started/ is a guide to get started for developing apps and clockface apps for Fitbit.
+
+<%= partial 'tutorials/_step-1-setup-free-account' %>
+
+h2(#create-clockface-app). Step 2 - Create a clock face 
+
+Prerequisites:
+
+* Make sure "Node and NPM are installed":https://nodejs.org/en/download/ on your system.
+* You will not need to install the Fitbit SDK separately as we'll be using NPX (that comes bundled with NPM) for doing this.
+* Fitbit user account. "Sign up here":https://www.fitbit.com/signup.
+* A Fitbit OS device, or the Fitbit OS Simulator for Windows or macOS.
+* The latest Fitbit mobile application for Android, iOS or Windows Phone, paired with your Fitbit device.
+* A computer with access to Fitbit Studio.
+* A Wifi connection to provide the Fitbit device a connection to the internet.
+
+We'll be using the Fitbit command line interface for creating the clockface. You can find the guide "here":https://dev.fitbit.com/build/guides/command-line-interface/ .
+
+We'll start by creating a new clockface. To do this, we'll be using the @create-fitbit-app@ command that allows you to generate the minimal scaffolding required for either an app or clock face.
+
+We'll call the app @train-app@. Let's create the app from the terminal by running:
+
+```[sh]
+npx create-fitbit-app train-app
+```
+It will prompt you with a few questions. In the first question @What type of application should be created?@ select the option @clockface@ . For the remaining questions you can move ahead with default.
+The new clockface app is ready, we need to open the @Fitbit OS Simulator@ and then run it locally as follows:
+
+```[sh]
+cd train-app
+npx fitbit
+bi
+exit
+```
+After running the clockface you should see a blank white screen on the simulator and the @Companion Settings@ window is empty currently with the title @App Settings@, to verify it has run successfully. 
+
+Let's install the package for @ably@, by running the following command from the terminal:
+
+```[sh]
+npm install --save ably
+```
+
+"See this step in Github":https://github.com/ably/tutorials/commit/195da37c041a90bb48f662228a3fc33498589178
+
+h2(#update-settings). Step 3 - Update the Settings
+
+Using the Companion Settings we take input from the user in a Fibtit clockface/app.
+So since we are making a clockface for the London Tube schedule, we ask user for three inputs the @Line@, @Station@ and @Towards@ .
+Each input is an autocomplete, hence the user can just type and select the option he/she desires. 
```

Comment:
An empty new line between these paragraphs would be good to make sure it renders nicely on the main site.